api_app/analyzers_manager/observable_analyzers/greynoiseintel.py

```diff
@@ -0,0 +1,58 @@
+# This file is a part of IntelOwl https://github.com/intelowlproject/IntelOwl
+# See the file 'LICENSE' for copying permission.
+
+from greynoise import GreyNoise
+
+from api_app.analyzers_manager import classes
+from api_app.exceptions import AnalyzerRunException
+from tests.mock_utils import MockResponse, if_mock_connections, patch
+
+
+class GreyNoiseAnalyzer(classes.ObservableAnalyzer):
+    def set_params(self, params):
+        self.api_version = params.get("greynoise_api_version", "v3")
+        self.max_records_to_retrieve = int(params.get("max_records_to_retrieve", 500))
+
+    def run(self):
+
+        if self.api_version == "v2":
+            api_key = self._secrets["api_key_name"]
+            session = GreyNoise(
+                api_key=api_key, integration_name="greynoise-intelowl-v1.0"
+            )
+            context_response = session.ip(self.observable_name)
+            riot_response = session.riot(self.observable_name)
```

Comment:
just to get a better understanding of this, could you please upload an example output of a successful query?

Reply:
I've added some screenshots to the issue, but essentially this merges the responses from both of our IP lookup endpoints and provides a consolidated output.

Reply:
thank you!